models.py

```diff
@@ -17,6 +18,14 @@
 TIME_FMT = "%Y-%m-%dT%H:%M:%S%z"
 
 
+class Status(Enum):
+    """Maintenance windows status."""
+
+    TO_RUN = 0
```

Comment:
Would it make sense to call it 'pending' instead of 'to_run'?

Reply:
Yes, it does. I'll make the change.